static/css/components/language.less

```diff
@@ -49,6 +49,9 @@
 .iaBar .language-component {
   position: relative;
   display: block;
+  border: 1px solid #999;
```

Comment:
See https://results.pre-commit.ci/run/github/69609/1726561025.TXkfKFmsRZGSgl_ArayurQ#:~:text=Unexpected%20hex%20color

We want to avoid using hex color references and instead use one of the existing registered colors from our https://github.com/internetarchive/openlibrary/blob/master/static/css/less/colors.less color file.

```
static/css/components/language.less
 52:21  ✖  Unexpected hex color "#999"  color-no-hex
```

The `donate` button in https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/site/alert.html uses a css treatment called `ghost-btn` to get its border here https://github.com/internetarchive/openlibrary/blob/97c252dd4b41db95c845a9ec713f84791d9cf461/static/css/components/buttonGhost.less#L4 which uses `@mid-grey`